src/riscv_opcodes/go_utils.py

```diff
@@ -9,12 +8,19 @@
 logging.basicConfig(level=logging.INFO, format="%(levelname)s:: %(message)s")
 
 
-def make_go(instr_dict: InstrDict):
-
-    args = " ".join(sys.argv)
-    prelude = f"""// Code generated by {args}; DO NOT EDIT."""
-
-    prelude += """
+def make_go(instr_dict: InstrDict, extensions: list[str]):
+    comment = """// Code generated with following arguments; DO NOT EDIT.
+/*
+make inst.go"""
+    if extensions:
+        comment += " EXTENSIONS='\\\n"
+        n = 4
+        exts = sorted(extensions)
+        for i in range(0, len(exts), 4):
+            ext_line = " ".join(exts[i : i + n])
+            comment += f"{ext_line:>30} \\\n"
```

Comment:
IMO, it would be better to have adjusting logic to support variable-length, instead of making it as a hardcoded limit